src/lightning_app/testing/testing.py

```diff
@@ -253,29 +260,35 @@ def run_app_in_cloud(
             env_copy["LIGHTNING_DEBUG"] = "1"
         shutil.copytree(app_folder, tmpdir, dirs_exist_ok=True)
         # TODO - add -no-cache to the command line.
-        process = Popen(
-            (
-                [
-                    sys.executable,
-                    "-m",
-                    "lightning",
-                    "run",
-                    "app",
-                    app_name,
-                    "--cloud",
-                    "--name",
-                    name,
-                    "--open-ui",
-                    "false",
-                ]
-                + extra_args
-            ),
-            cwd=tmpdir,
-            env=env_copy,
-            stdout=sys.stdout,
-            stderr=sys.stderr,
-        )
-        process.wait()
+        stdout_path = get_logfile("running_process")
+        with open(stdout_path, "w") as stdout:
+            cmd = [
+                sys.executable,
+                "-m",
+                "lightning",
+                "run",
+                "app",
+                app_name,
+                "--cloud",
+                "--name",
+                name,
+                "--open-ui",
+                "false",
+            ]
+            process = Popen((cmd + extra_args), cwd=tmpdir, env=env_copy, stdout=stdout, stderr=sys.stderr)
+            process.wait()
+
+        if is_editable_mode:
```

Comment:
I would make this condition always true on the CI, perhaps like:

```suggestion
        if is_editable_mode or PACKAGE_LIGHTNING :
```

Reply:
I see, I'm just worried we have this silently changed in the future and not erroring